### ISSUE TITLE
Provide descriptions for default user roles

### ISSUE THREAD

**Issue by MartinWahnschaffe** [SORMAS-Foundation/SORMAS-Project#8945]
### Problem Description
User's currently have to rely on external documentation to understand what the default user roles are about.
Should be done after #4461 is finalized.

### Proposed Change
- [ ] Add descriptions to default user roles using I18nProperties.getEnumDescription
- [ ] When creating user roles based on the default user role on startup copy the description into the new user role
- [ ] If #4463 is already done: In the UI for creating new user roles, fill the description with the description of the selected user template (default user role or existing user role). Otherwise move this todo over to #4463
- [ ] Display the description of the user role (not the default user role) for each role in the UserEditForm

### Additional Information
@kwa20 Please provide the english descriptions for all default user roles.
